tests/util/benchmark.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
Yes, this comes from a shellcheck suggestion, using the `$SECONDS` bashism requires it.